### ISSUE TITLE
allowOverResize

### ISSUE THREAD

**Issue by SMen1** [albanx/real-uploader#1]
Hi
I have set the allowOverResize 
resizeImage: {
                maxWidth: 3500,
                maxHeight: 4500,
                scaleMethod: 3,
		allowOverResize: true,
		outputQuality:.2,
            },

My expectation is that an image with width less than 3500 would be processed  with this option set to true. Is this correct? At the moment Images smaller than the maximum do not appear to be being processed when this is set to true.

Many thanks
Simon

**Comment by SMen1** [comment]
[test.txt](https://github.com/albanx/real-uploader/files/1368372/test.txt)
this is the full code

**Comment by albandaft** [comment]
With `allowOverResize` option to true, the image should scale up (stretch) even it is smaller than the give maxWidth/Height. Example if you have an image of 350x450 it will scape up to 3500x4500, of course pixelated

**Comment by SMen1** [comment]
I am observing this:

The upload starts, but the file is not reduced in size, so if the image is 2 mbytes then this is being uploaded full size.
I am then finding that the image upload stops at 100% and does not complete, but on the server I can see the thumb and the uploaded file. The uploaded file has been reduced in size, but the upload was slow suggesting things have been done in reverse order - upload then resize.

I have been experimenting with a listener on preview to try and get the image size to adjust the maxWidth parameter if the image is less, but not sure how to do this.
The upload then

**Comment by SMen1** [comment]
Yes the image is hanging still at 100% with the allowOverResize set to true. Also the reduced file size is greater than the setting I have for quality. (I have tested this by resizing the image to my maximum and then uploading with my quality setting).

So it seems that a different quality setting is being used as default - though I am guessing obviously.

**Comment by albandaft** [comment]
OK I will have a look at this. Right now I am working on the ASPX script

**Comment by SMen1** [comment]
OK thank you, I will be trying more things tomorrow.
